diff --git a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.kt b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.kt
index bb945fe..a1110d8 100644
--- a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.kt
+++ b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.kt
@@ -25,7 +25,7 @@ class ExoPlayerView @JvmOverloads constructor(context: Context, attrs: Attribute
     private var localStyle = SubtitleStyle()
     private var pendingResizeMode: Int? = null
     private val liveBadge: TextView = TextView(context).apply {
-        text = "LIVE"
+        text = ""
         setTextColor(Color.WHITE)
         textSize = 12f
         val drawable = GradientDrawable()
@@ -215,7 +215,7 @@ class ExoPlayerView @JvmOverloads constructor(context: Context, attrs: Attribute
         val seekable = player.isCurrentMediaItemSeekable
 
         // Show/hide badge
-        liveBadge.visibility = if (isLive) View.VISIBLE else View.GONE
+        liveBadge.visibility = if (isLive) View.GONE else View.GONE
 
         // Disable/enable scrubbing based on seekable
         val timeBar = playerView.findViewById<DefaultTimeBar?>(androidx.media3.ui.R.id.exo_progress)
